### DISCUSSION TITLE
Detail deployment configuration and process when deploying agent with gateway architecture

### DISCUSSION THREAD

**Discussion by sudotechie** [signalfx/splunk-otel-collector#2136] [General]
For companies that would like to deploy the agent + gateway architecture, there are a few gaps that I've seen:

1. Within the configs, there is no comment indicating whether the collector is running in gateway or agent mode. A user would have to know that host metrics are not scraped when running as gateway or that passthrough is not present
2. I don't see where it is clearly defined what data is not available for the host that the gateway runs on, if deployed in Linux vs. K8s
3. Are there any ordering requirements or variables that must be present for a gateway that do not apply for an agent?

**Comment by atoulme** [comment]
We provide an agent and gateway config - would those help shed some light? What type of documentation and expectation would you like to explore?

Please take a look here:
https://github.com/signalfx/splunk-otel-collector/tree/main/cmd/otelcol/config/collector
